dom.bs

```diff
@@ -6965,8 +6965,16 @@ a boolean <var>delegatesFocus</var>, and a boolean <var>slotAssignment</var>:
    is false, then <a>throw</a> an "{{NotSupportedError!!exception}}" {{DOMException}}.
 
    <li>
-    <p>Otherwise, <a for=/>remove</a> all of <var>element</var>'s <a for=Element>shadow root</a>'s
-    <a for=tree>children</a>, in <a>tree order</a>, and return.
+    <p>Otherwise:
+
+    <ol>
+      <li><p><a for=/>Remove</a> all of <var>element</var>'s <a for=Element>shadow root</a>'s
```

Comment:
Single-space indentation.

Reply:
Done. Context switching is hard - Blink is 2-spaces. 😄